addons/website/static/src/snippets/s_image_gallery/000.js

```diff
@@ -71,7 +71,17 @@ const GalleryWidget = publicWidget.Widget.extend({
                 editableMode: false,
                 $target: self.$modal.find('.modal-body.o_slideshow'),
             });
+            self.$modal[0].addEventListener("keydown", self._onArrowKey.bind(self));
         });
+        this.$modal.one("hide.bs.modal", () => {
+            this.$modal[0].removeEventListener("keydown", this._onArrowKey);
```

Comment:
It is not correct to remove the event listener like this because you need to remove the exact same handleras when you added the listener, so you need to save it somewhere and use this saved one when adding/removing the listeners :wink: 

Here to make it more simple,  you could just create a variable before the `this.$modal.one('shown.bs.modal', function () {` line 
like this:
```js
const onModalKeydown = (ev) => {
  // Your handler code
}
```
and use this variable 
```js
self.$modal[0].addEventListener("keydown", onModalKeydown);
...
this.$modal[0].removeEventListener("keydown", onModalKeydown);
```

It will avoid having to play with the `bind` and all :slightly_smiling_face: 

Or if you want to keep the handler like now, just save it in `this` :wink:

Reply:
I opted for a double underscore method as is done elsewhere in the code.